syft/frameworks/torch/nn/functional.py

```diff
@@ -98,12 +105,22 @@ def _pre_conv(input, weight, bias=None, stride=1, padding=0, dilation=1, groups=
     nb_channels_out, nb_channels_kernel, nb_rows_kernel, nb_cols_kernel = weight.shape
 
     if bias is not None:
-        assert len(bias) == nb_channels_out
+        # assert len(bias) == nb_channels_out
+        if len(bias) != nb_channels_out:
+            raise ValueError("Check size of bias")
 
     # Check if inputs are coherent
-    assert nb_channels_in == nb_channels_kernel * groups
-    assert nb_channels_in % groups == 0
-    assert nb_channels_out % groups == 0
+    # assert nb_channels_in == nb_channels_kernel * groups
+    # assert nb_channels_in % groups == 0
+    # assert nb_channels_out % groups == 0
+    if nb_channels_in != nb_channels_kernel * groups:
+        raise ValueError("Given inputs are not supported")
```

Comment:
Q: Could you specify here given inputs are not supported for ```nb_channels_in``` - the same for the ones bellow